src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreTargetGraph.cs

```diff
@@ -17,52 +17,48 @@ public class RestoreTargetGraph : IRestoreTargetGraph
         /// <summary>
         /// Gets the runtime identifier used during the restore operation on this graph
         /// </summary>
-        public string RuntimeIdentifier { get; internal set; }
+        public string RuntimeIdentifier { get; }
```

Comment:
Given that there's so many changes in this file, can we nullable annotate this now?
It'll be super helfpul

Reply:
I think we should tackle that in a different change

Reply:
We tend to take a best effort approach towards nullability when the type is changed a lot, which I'd say in this case is a bunch of changes. 
I'd prefer if it was done in this PR, but I'm not gonna block this PR on that. 
Your call.